sydent/replication/peer.py

```diff
@@ -241,6 +241,10 @@ def _failedPushBodyRead(self, body, updateDeferred):
         :param updateDeferred: The deferred to call the error callback of.
         :type updateDeferred: twisted.internet.defer.Deferred
         """
+        # json.loads doesn't allow bytes in Python 3.5
+        if isinstance(body, bytes):
```

Comment:
according to the docstring, it can't be a bytes (at least on python 3), so either the docstring is lying or this is redundant.